tests/postgres_tests/test_json.py

```diff
@@ -356,6 +359,9 @@ def test_iexact(self):
     def test_icontains(self):
         self.assertFalse(JSONModel.objects.filter(field__foo__icontains='"bar"').exists())
 
+    def test_nested_icontains(self):
+        self.assertTrue(JSONModel.objects.filter(field__d__icontains='e').exists())
```

Comment:
I've checked and `icontains` is not affected, we can remove this test. Sorry I should checked this first.

Reply:
So should I remove the `icontains` test? Even if it's not affected, could it be good to have a unit test on this ORM call?

Reply:
This use case is already tested in `test_icontains()`.